admin_apikey.proto

```diff
@@ -0,0 +1,61 @@
+syntax = "proto3";
+
+import "status.proto";
+
+option go_package = "vectara.com/public/proto/admin";
+
+option java_package = "com.vectara.admin";
+option java_outer_classname = "AdminApiKeyProtos";
+
+package com.vectara.admin;
+
+/******************************************************************************
+  API Keys
+ ******************************************************************************/
+// Types of ApiKey.
+enum ApiKeyType {
+  API_KEY_TYPE__UNDEFINED = 0;
+  // ApiKey for serving. Only gives access to query data.
+  API_KEY_TYPE__SERVING = 1;
+  // ApiKey for serving and indexing. Gives access to both query and index data.
+  API_KEY_TYPE__SERVING_INDEXING = 2;
+}
+
+// Status of ApiKey.
+enum ApiKeyStatus {
+  UNKNOWN = 0;
+  ENABLED = 1;
+  DISABLED = 2;
+  DELETED = 3;
+}
+
+
+// This request can be used to create one or more ApiKeys.
```

Comment:
"Use this request to..."

Reply:
Hey @pwoznic , can  you approve without this change. I will update this in a later PR. The journey to update is quite long, as I have to update it in bitbucket platform code, then run the pipeline to push it to this repo's dev branch, and then merge the dev branch again. 
Since this is a small change, we can handle this in a follow up PR. wdyt?

Reply:
> Hey @pwoznic , can you approve without this change. I will update this in a later PR. The journey to update is quite long, as I have to update it in bitbucket platform code, then run the pipeline to push it to this repo's dev branch, and then merge the dev branch again. Since this is a small change, we can handle this in a follow up PR. wdyt?

Sure, sounds good!